spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/JdbcOutboundGateway.java

```diff
@@ -172,6 +174,7 @@ public void setReplySqlParameterSourceFactory(SqlParameterSourceFactory sqlParam
 	 * @param rowMapper the {@link RowMapper} to use.
 	 */
 	public void setRowMapper(RowMapper<?> rowMapper) {
+		Assert.notNull(this.poller, "If you want to set 'rowMapper', then you must provide a 'selectQuery'.");
```

Comment:
The technical documentation and similar messages must be impersonated.
So, we cannot use words like `you`, `we`, `they` etc. in these error messages.